python/tank/authentication/saml2_sso.py

```diff
@@ -0,0 +1,581 @@
+# Copyright (c) 2017 Autodesk.
```

Comment:
Agreed... There would be a common package, and then client specific integration files (RV support events, not toolkit, etc.)